app/controllers/admin/claim_checks_controller.rb

```diff
@@ -22,6 +25,10 @@ def load_claim
     @matching_claims = Claim::MatchingAttributeFinder.new(@claim).matching_claims
   end
 
+  def claims_preventing_payment_finder
+    @claims_preventing_payment_finder ||= Claim::ClaimsPreventingPaymentFinder.new(@claim)
```

Comment:
Given this is used in all sorts of other places, I wonder if `claims_preventing_payment_finder` could be a method in the model?

Reply:
I'm not sure it's an appropriate thing to memoize on the model, since it can be invalidated by e.g. deleting another claim. So then it would just be a method on the model which instantiates this object and returns it, and I'm not sure it's worth it?

Reply:
Yeah, fair enough 👍